control_panel_api/settings/base.py

```diff
@@ -167,6 +167,9 @@ def is_enabled(value):
 RSTUDIO_AUTH_CLIENT_ID = os.environ.get('RSTUDIO_AUTH_CLIENT_ID')
 RSTUDIO_AUTH_CLIENT_SECRET = os.environ.get('RSTUDIO_AUTH_CLIENT_SECRET')
 
+ELASTICSEARCH_CONN = os.environ.get('ELASTICSEARCH_CONN')
```

Comment:
Let's not forget to add this config to the helm chart/config or it will blow up 💥 😄

Reply:
😀